input/pagecontent/changes.md

```diff
@@ -86,11 +86,12 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from  Condition.recorder [FHIR-45019](https://jira.hl7.org/browse/FHIR-45019), [FHIR-45082](https://jira.hl7.org/browse/FHIR-45082)
   - removed Must Support from Condition.encounter [FHIR-45189](https://jira.hl7.org/browse/FHIR-45189)
   - added profile-specific implementation guidance on representing no known problems [FHIR-45058](https://jira.hl7.org/browse/FHIR-45058)
-- Removed Must Support from the following elements in AU Core AllergyIntolerance:
-  - AllergyIntolerance.asserter [FHIR-44699](https://jira.hl7.org/browse/FHIR-44699), [FHIR-45083](https://jira.hl7.org/browse/FHIR-45083)
-  - AllergyIntolerance.recorder [FHIR-45082](https://jira.hl7.org/browse/FHIR-45082), [FHIR-45066](https://jira.hl7.org/browse/FHIR-45066)
-  - AllergyIntolerance.encounter [FHIR-45123](https://jira.hl7.org/browse/FHIR-45123)
-  - AllergyIntolerance.note [FHIR-45188](https://jira.hl7.org/browse/FHIR-45188)
+- Made the following changes to AU Core AllergyIntolerance:
+  - removed Must Support from AllergyIntolerance.asserter [FHIR-44699](https://jira.hl7.org/browse/FHIR-44699), [FHIR-45083](https://jira.hl7.org/browse/FHIR-45083)
+  - removed Must Support from AllergyIntolerance.recorder [FHIR-45082](https://jira.hl7.org/browse/FHIR-45082), [FHIR-45066](https://jira.hl7.org/browse/FHIR-45066)
+  - removed Must Support from AllergyIntolerance.encounter [FHIR-45123](https://jira.hl7.org/browse/FHIR-45123)
+  - removed Must Support from AllergyIntolerance.note [FHIR-45188](https://jira.hl7.org/browse/FHIR-45188)
+  - changed the AllergyIntolerance.code binding Indicator of Hypersensitivity or Intolerance to Substance to be extensible, and added Adverse Reaction Substances and Negated Findings as an Additional Binding with a purpose as candidate [FHIR-44713](https://jira.hl7.org/browse/FHIR-44713) 
```

Comment:
1. Say "value set" after "Indicator of ...".
1. Split the "and" into a separate bullet point, and make them both sub bullet of "changed AllergyIntolerance.code":
1. Instead of the existing "Added Adverse Reaction Substances..." say "Added Additional Binding, Adverse Reactions Substances and Negated Findings value set as candidate."

Reply:
Changed.